sparts/vtask.py

```diff
@@ -201,6 +201,7 @@ def set_exception(self, exception):
         self.timer.stop()
         if self.future is not None:
             self.future.set_exception(exception)
+            handled = True
```

Comment:
Will do - thanks! In case of QueueTask - this decision makes sense since submit() returns a future instance so it's basically a part of it's public API.

Also in QueueTask.map() we rely on the future to throw to the caller as we call result() on it.